src/Console/ConfigurationResolver.php

```diff
@@ -282,6 +285,8 @@ public function getDiffer()
                         $option
                     ));
                 }
+            } elseif (RawFileOutputReporter::NAME === $this->options['format']) {
+                $option = 'raw';
```

Comment:
so, it is possible to use this reporter when someone is fixing full dir on drive.
how it will behave?

Reply:
check `doc/raw.md`